app.js

```diff
@@ -0,0 +1,26 @@
+const bar = document.querySelectorAll('.bar');
```

Comment:
Great Job having your javascript codes in a seperate file. Makes your code more readable.